src/Navigation/Resumedownload/index.css

```diff
@@ -0,0 +1,52 @@
+.resume-download {
+  padding: 20px;
+  display: flex;
+  flex-direction: row;
+  background-color: var(--primary-color);
+  opacity: 0.9;
+  justify-content: center;
+}
+.resume-download_logo {
+  background-image: url(../../images/download1.png);
+  width: 130px;
+  height: 130px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.resume-download_heading {
+  font-size: 38px;
```

Comment:
That is nice that you have made heading-color a constant. I feel heading font-size should be a constant too, to be used in all other section headings.
What do you think?